src/webview/html/messageHeaderAsHtml.js

```diff
@@ -29,22 +31,23 @@ export default (
     return '';
   }
 
-  if (isStreamNarrow(narrow)) {
+  if (isStreamNarrow(narrow) || isTopicNarrow(narrow)) {
     const topicNarrowStr = JSON.stringify(topicNarrow(item.display_recipient, item.subject));
     const topicHtml = renderSubject(item);
 
     return template`
 <div
-  class="header-wrapper header topic-text"
+  class="header-wrapper header topic-content"
```

Comment:
This name is kind of confusing -- we generally use "content" for the content of the message itself.

Perhaps `topic-header`?